wasm2c/wasm-rt-impl.c

```diff
@@ -42,6 +42,42 @@
 #define alloca _alloca
 #endif
 
+#if !defined(_WIN32)
+
+#include <pthread.h>
+#define WASM_RT_MUTEX_TYPE pthread_mutex_t
+#define WASM_RT_CREATE_MUTEX(lock) \
+  do {                             \
+  } while (0)
+#define WASM_RT_DESTROY_MUTEX(lock) \
+  do {                              \
+  } while (0)
+#define WASM_RT_MUTEX_LOCK(lock)        \
+  if (pthread_mutex_lock(&(lock)) != 0) \
+  abort()
```

Comment:
Does clang-format not indent the abort calls here?

Reply:
It removes the indent unfortunately

Reply:
Adding a semi-colon after the abort and wrapping the whole thing in a do-while fixes the clang-formatter. Will make this change